src/contracts/core/StrategyManagerStorage.sol

```diff
@@ -63,9 +63,8 @@ abstract contract StrategyManagerStorage is IStrategyManager {
 
     /*
      * Reserved space previously used by the deprecated mapping(address => uint256) beaconChainETHSharesToDecrementOnWithdrawal.
-     * This mapping tracked beaconChainETH "debt" in case updates were made to shares retroactively.  However, this design was
-     * replaced by a simpler design that prevents withdrawals from EigenLayer before withdrawals from the beacon chain, which
-     * makes this tracking unnecessary.
+     * This mapping tracked beaconChainETH "deficit" in cases where updates were made to shares retroactively.  However, this construction was
```

Comment:
take out `withdrawalDelayBlocks`?

Reply:
`numWithdrawalsQueued` unused as well

Reply:
I tried changing these, but Alex pointed this out as well https://github.com/Layr-Labs/eigenlayer-contracts/pull/222#discussion_r1348947449
Take a look at the new comments. I think reverting the names but marking these as 'internal' and noting the deprecation in the comments is probably the best move here.

Reply:
switched back to old naming here https://github.com/Layr-Labs/eigenlayer-contracts/pull/222/commits/f1ebd25d35edbf213ed3d79d75bf8ef12a3a6fbd. I think the comments are sufficient about deprecation, but please flag anything that's a bit lacking on clarity (or just make a quick commit to clarify the deprecation).